### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by MuhammadUsman94**

### CODE REVIEW

**Comment by brucker3** [comment]
the code looks good can you wrape it into a class tho. 
also it should be in the CS5551/mysite/checkers I could not tell if that was the directory it was under

**Review by zpatel-dev — DISMISSED** [review]
I would suggest putting these files in one repo so it would be easier in the future to organize

File: Checker_board.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
+w, h = 8, 8;
```
Comment (brucker3) [inline Checker_board.py]: is this checker_board.py needed for the actual pr or is this still code your working on

File: board.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+
```
Comment (brucker3) [inline board.py:1]: this class looks good. 
where is this file in the directory structure?
it should be in CS5551/mysite/checkers

File: Checker_board.py (hunk 1 above)
Comment (MuhammadUsman94) [reply Checker_board.py]: No, checker_board.py im not working on that. May be i would need to delete it from here

File: board.py (hunk 2 above)
Comment (MuhammadUsman94) [reply board.py:1]: I don’t know about the file directory structure. Simply i put it in my branch that is usman_work.

**Pull request closed without merging**